wikimedia/wikipedia-ios-components
# Add project icons

**Phabricator:** https://phabricator.wikimedia.org/T342938

### Notes
* This PR adds SwiftUI project icons to the components package

### Test steps
* Run the demo app, click on Project Icons
* Make sure the theming works
